incubator/kong/README.md

```diff
@@ -0,0 +1,142 @@
+# Kong
+
+[Kong](https://getkong.org/) is a scalable, open source API Layer (also known as an API Gateway, 
+or API Middleware). Kong runs in front of any RESTful API and is extended through Plugins, which 
+provide extra functionality and services beyond the core platform. See 
+[What is Kong?](https://getkong.org/about/) for more information.
+
+__DISCLAIMER:__ This Helm Chart was created using Kong documentation, but it is not an officially 
+supported or maintained Mashape product.
+
+## Introduction
+
+This Chart bootstraps Kong as a Kubernetes deployment with support for proxy and admin endpoints.
+
+__NOTE:__ This Chart can be configured to use Kong supported data stores of PostgreSQL or Cassandra
+but it does not intend to manage or deploy those sources. See the 
+[PostgreSQL Chart](https://github.com/kubernetes/charts/tree/master/stable/postgresql) 
```

Comment:
@viglesiasce any ideas how I get access to the secret [generated here](https://github.com/kubernetes/charts/blob/master/stable/postgresql/templates/secrets.yaml#L15)?